garage/tf/q_functions/discrete_cnn_q_function.py

```diff
@@ -0,0 +1,190 @@
+"""Discrete MLP QFunction."""
```

Comment:
We should state this CNN network actually supports CNN2MLP. This is not the same as the CNN* primitive in the current garage. If you think the naming is ok, please add more details to this documentation.